Examples/ObjectiveC/ExtrusionsExample.m

```diff
@@ -21,7 +21,7 @@ @implementation ExtrusionsExample
 - (void)viewDidLoad {
     [super viewDidLoad];
     
-    MGLMapView *mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds styleURL:[MGLStyle lightStyleURLWithVersion:9]];
+    MGLMapView *mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds styleURL:[MGLStyle lightStyleURL]];
```

Comment:
This example relies on details about the style’s existing source. We should keep specifying the version in such cases.